docs/zh/guide/advanced/expand.md

```diff
@@ -197,3 +199,277 @@ public enum FrameworkEnum {
 开发流程就是这样，主要的难点在于`IDocBuildTemplate`的实现处理。
 
 
+## 其他框架文档解析开发（基于`JAVA SPI`） `@since 3.0.6`
+:::tip
+`@since 3.0.6` 开始，支持不修改源码的情况下，新增其他框架文档的解析🎉
+:::
+
+目前支持`Spring`技术栈`Web`、`Apache 
+Dubbo`层面的解析。由于官方开源人力有限，因此无法去满足解析其他的`web`层框架。
+当然要`Web`层面的框架，一般需要框架需要满足下面的条件：
+- 框架使用明确的注解路由(通俗就是说类似`Spring`的`Controller`有明确的注解申明`path`路径)，也可以是类似`Jakarta RS-API 2.x`规范的实现框架。
+
+
+下面来看下实现支持编写。
+
+### 方案一：插件引入开发
+在maven插件引入对应的扩展依赖完成扩展。
+#### 核心代码编写
+新增一个模块或项目，引入`smart-doc`依赖
```

Comment:
Here, you can change it to: “新增一个smart-doc的解析扩展项目或模块，用于扩展解析一些自定义的框架，在扩展项目中引入smart-doc官方的依赖。对于企业内部框架通用解析，推荐采用独立项目来构建扩展解析能力。”  This can better guide users to build a universal and standardized extension.